mandatory/thinking-like-a-programmer/task.md

```diff
@@ -2,7 +2,7 @@
 
 ## 1. Install ESLint
 
-ESLint is what is called an static analysis tool. It checks your code for common mistakes that can impact code quality and styling. 
+ESLint is what is called an static analysis tool. It checks your code for common mistakes that can impact code quality and styling.
```

Comment:
Great that you have installed ESList!

You may aware that you have committed all contents under **node_module** folder which is undesirable. Let learn how to ignore files under a git repository by using **.gitignore**